faq.md

```diff
@@ -141,44 +141,30 @@ we encourage you to email our mailing list:
 
 ## Can I access native services and APIs like sensors and local storage?
```

Comment:
Probably best to avoid the word 'native'. Use 'platform' or 'host' instead. Several places.